src/main/java/ru/skypro/homework/controller/AdsController.java

```diff
@@ -0,0 +1,66 @@
+package ru.skypro.homework.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import ru.skypro.homework.dto.Ad;
+import ru.skypro.homework.dto.Ads;
+import ru.skypro.homework.dto.CreateOrUpdateAd;
+import ru.skypro.homework.dto.ExtendedAd;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/ads")
+public class AdsController {
+
+    //здесь будет зависимость от AdsService
+
+    @GetMapping("/")
+    public ResponseEntity<Ads> getAllAds() {
+        //здесь будет вызов сервиса
+        return new ResponseEntity<>(new Ads(), HttpStatus.OK);
+    }
+
+    @PostMapping(value = "/", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
+    public ResponseEntity<Ad> postAd(@RequestBody CreateOrUpdateAd properties,
```

Comment:
```suggestion
    public ResponseEntity<Ad> postAd(@RequestParam CreateOrUpdateAd properties,
```
иначе не будет работать